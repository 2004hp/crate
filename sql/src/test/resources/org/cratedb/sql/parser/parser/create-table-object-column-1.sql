CREATE TABLE t1 (col1 object)