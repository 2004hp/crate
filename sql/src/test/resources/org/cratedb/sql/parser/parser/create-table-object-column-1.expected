org.cratedb.sql.parser.parser.CreateTableNode@69edfb80
name: t1
statementType: CREATE TABLE
lockGranularity: R
existenceCheck: NO_CONDITION
tableElementList: 	
	org.cratedb.sql.parser.parser.TableElementList@71ac6d5

	[0]:	
	org.cratedb.sql.parser.parser.ObjectColumnDefinitionNode@6ef3cd97
	type: OBJECT
	name: col1
	elementType: null
	objectType: DYNAMIC
	subColumns: 		
		org.cratedb.sql.parser.parser.TableElementList@1648a4ad
